packages/@react-spectrum/s2/src/Calendar.tsx

```diff
@@ -304,11 +305,17 @@ export const Calendar = /*#__PURE__*/ (forwardRef as forwardRefType)(function Ca
       {({isInvalid, isDisabled}) => {
         return (
           <>
-            <Header styles={headerStyles}>
-              <CalendarButton slot="previous"><ChevronLeftIcon /></CalendarButton>
-              <CalendarHeading />
-              <CalendarButton slot="next"><ChevronRightIcon /></CalendarButton>
-            </Header>
+            <Provider
```

Comment:
we already clear some context in https://github.com/adobe/react-spectrum/blob/863ac427ea3332ae7cd99ccaf2ad1c92a03c6545/packages/%40react-spectrum/s2/src/DatePicker.tsx#L259
should we clear these there as well? then we don't need to do it in two files

Reply:
I guess maybe? it might pick up some stray context, but same argument could be made for the overlay trigger state context

Reply:
they'd have the same issue with the overlaytriggerstatecontext too, so that should probably be moved to the same place